### ISSUE TITLE
NPE in ScanBaseActivity.onClick()

### ISSUE THREAD

**Issue by praveeng145** [getbouncer/cardscan-android#80]
Crash log - 
```
Fatal Exception: java.lang.NullPointerException
Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
keyboard_arrow_up
arrow_right
com.getbouncer.cardscan.base.ScanBaseActivity.onClick (ScanBaseActivity.java:354)
```

As per Android docs [here](https://developer.android.com/reference/android/hardware/Camera.Parameters.html#getFlashMode()) -  `Camera.Parameters.getFlashMode()` can be null. 

```
current flash mode. null if flash mode setting is not supported.
```

We are using `com.getbouncer:cardscan:1.0.5001` version but this crash can happen on the latest version as well because the master branch don't have a null-check [here](https://github.com/getbouncer/cardscan-android/blob/master/cardscan-base/src/main/java/com/getbouncer/cardscan/base/ScanBaseActivity.java#L508-L509)

Let me know if you need more details.

**Comment by awushensky** [comment]
Thanks for finding this, I'm working on fixing it now.

**Comment by awushensky** [comment]
A fix has been released as part of 1.0.5122. Thank you for reporting!
